.ruby-gemset

```diff
@@ -0,0 +1 @@
+create_register_picker
```

Comment:
I am so bad at remembering the new line rule. Make sure you call it out on mine too when you see it.